packages/@aws-cdk/aws-dynamodb/lib/table.ts

```diff
@@ -13,27 +15,24 @@ const RANGE_KEY_TYPE = 'RANGE';
 // https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Limits.html#limits-secondary-indexes
 const MAX_LOCAL_SECONDARY_INDEX_COUNT = 5;
 
-const READ_DATA_ACTIONS = [
-  'dynamodb:BatchGetItem',
-  'dynamodb:GetRecords',
-  'dynamodb:GetShardIterator',
-  'dynamodb:Query',
-  'dynamodb:GetItem',
-  'dynamodb:Scan',
-];
-
-const READ_STREAM_DATA_ACTIONS = [
-  'dynamodb:DescribeStream',
-  'dynamodb:GetRecords',
-  'dynamodb:GetShardIterator',
-];
-
-const WRITE_DATA_ACTIONS = [
-  'dynamodb:BatchWriteItem',
-  'dynamodb:PutItem',
-  'dynamodb:UpdateItem',
-  'dynamodb:DeleteItem',
-];
+/**
+ * What kind of server-side encryption to apply to this table.
+ */
+export enum TableEncryption {
+  /**
+   * Server-side KMS encryption with a customer master key managed by customer.
+   * If `encryptionKey` is specified, this key will be used, otherwise, one will be defined.
+   */
+  CUSTOMER_MANAGED = 'CUSTOMER_MANAGED',
+  /**
+   * Server-side KMS encryption with a master key managed by AWS.
+   */
+  AWS_MANAGED = 'AWS_MANAGED',
+  /**
+   * Server-side KMS encryption with a master key owned by AWS.
+   */
+  DEFAULT = 'AWS_OWNED'
```

Comment:
I wonder if there's a better name for this than `DEFAULT`, but I also think `AWS_OWNED` is not a great name either; and I fail to come up with anything better... Oh well.

Reply:
In AWS Console, "DEFAULT" is used for AWS owned CMK. https://aws.amazon.com/blogs/database/bring-your-own-encryption-keys-to-amazon-dynamodb/

Reply:
That makes it a sensible name :) Thanks for pointing to it!